test/unit/authorize-suite.js

```diff
@@ -138,6 +138,44 @@ define(['requirejs', 'fs'], function(requirejs, fs, undefined) {
       },
 
       {
+        desc: "_rs_init extracts rsDiscovery data param for configuring the WireClient",
+        run: function(env, test) {
+          var storage = new RemoteStorage();
+
+          // mock the atob function, as it's not available in node
```

Comment:
How is this handled when rs.js is run in node? Wouldn't it be polyfilled in the source?